azureblob/azureblob_test.go

```diff
@@ -0,0 +1,132 @@
+package azureblob
+
+import (
+	"github.com/Azure/azure-sdk-for-go/sdk/storage/azblob/bloberror"
+	"github.com/gofiber/fiber/v2/utils"
+	"testing"
+)
+
+func newStore() *Storage {
+	return New(Config{
+		Account:   "azurite",
+		Container: "test",
+		Endpoint:  "http://127.0.0.1:10000/azurite",
+		Credentials: Credentials{
+			Account: "azurite",
```

Comment:
We should probably use the default storage account here: 

Source: https://github.com/Azure/Azurite#default-storage-account

Reply:
Fixed